packages/framework/src/Model/Product/Search/FilterQuery.php

```diff
@@ -70,67 +70,66 @@ public function applyOrdering(string $orderingModeId, PricingGroup $pricingGroup
     {
         $clone = clone $this;
 
+        $clone->sorting = [];
+        $clone->sorting['availability_dispatch_time'] = 'asc';
```

Comment:
I have mind problem with this as default for all orderings. I think, it should be applied only for default ordering but not for all. Consider I am ordering alphabetically from A to Z and I am looking for Apple iPhone 13 Pro, but it is not on stock yet, because it just has been released and manufacturer cannot keep up with demand, so currently it is out of stock and I do not see it under A, because it is somewhere on 13th page after on stock products with Z at beginning of name.

What do you think @pk16011990 ?

Reply:
this is bussiness decision